src/utilities.jl

```diff
@@ -419,7 +419,7 @@ function keywords(func, m::Method)
     # table is a MethodTable object. For some reason, the :kwsorter field is not always
     # defined. An undefined kwsorter seems to imply that there are no methods in the
     # MethodTable with keyword arguments.
-    if isdefined(table, :kwsorter)
+    if  !hasfield(Core.MethodTable, :kwsorter) || isdefined(table, :kwsorter)
```

Comment:
Julia 1.0 support, I'll probably try get around to dropping 1.0 support relatively soon.

```suggestion
    if  !(fieldindex(Core.MethodTable, :kwsorter, false) > 0) || isdefined(table, :kwsorter)
```